### PR TITLE
Created Journal Week 1

### PR DESCRIPTION
**Pull request by karjac14**

### CODE REVIEW

File: journal-week-1.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# One Down Looking Forward for Eleven More 
+
+
+The first week was a shock, I indeed entered a boot camp. No hanky panky. Starting from the prep-work and the rest of the week, all of us from the class had the taste of brain squeezing experience.
```
Comment (kellymurray) [inline journal-week-1.md]: "Hanky Panky" _could_ be seen as a loaded term - I can explain in lab if you like. I would not use it here though. :) If you need help brainstorming more phrases, come see me.

Comment (karjac14) [inline journal-week-1.md]: @kellymurray, uh oh, i just goggled it and saw the meaning, we have different meaning of that in our hometown. will definitely delete it though.

**Comment by ericquayson** [comment]
:+1: 

I felt the same way too! Keep pushing! 

It's hard sometimes to see new abstract concepts for the first time. However, more practice, reading, and writing javascript will help with that.

**Comment by nickycadavillo** [comment]
:+1:  Great entry!  Your writing was great and informative. I too know exactly how you felt in the beginning.  We're all in this together!

File: journal-week-1.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+* Javascript - we learned the foundation this language, starting from the basic types of objects and manipulating these to form simple apps.
+
+* Creating and Testing JS Files - we learned to test our JS files, using some codes in the terminal such as `node`, `mocha` and `chai`. These test frameworks had given me different perspective in testing my js apps other than the conventional console.
+ 
+* Talk & Think like a developer - the first week gave me gave me the idea on what my life as Developer will be. Staring at codes the whole day. haha. Well I honestly enjoyed coding and I get excited when my expected outputs are starting to show up.
+
+* The great growth Mindset - In relation to the hardships we had in the first week, It's great that the growth mindset was introduced to us. Applying this mindset reinforced us to survive and convert our wounds to battle scars. :sunglasses:
+
+From the experience, I awaken my brain from someone who do normal 9-5  office mindset to a front end rising action hero. Looking forward for the next 11 weeks of challenges and fulfillment.
```
Comment (kellymurray) [inline journal-week-1.md:18]: :muscle:

File: journal-week-1.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+In particular, these are the things we learned in a short span of one week:
+
+* Github - we created repos in this wonderful version control system. From here we can share, fork our working files. We also learned how to sync my files to our local computer to work with the files and then push it back to Github again,
+
+* Javascript - we learned the foundation this language, starting from the basic types of objects and manipulating these to form simple apps.
+
+* Creating and Testing JS Files - we learned to test our JS files, using some codes in the terminal such as `node`, `mocha` and `chai`. These test frameworks had given me different perspective in testing my js apps other than the conventional console.
+ 
+* Talk & Think like a developer - the first week gave me gave me the idea on what my life as Developer will be. Staring at codes the whole day. haha. Well I honestly enjoyed coding and I get excited when my expected outputs are starting to show up.
```
Comment (kellymurray) [inline journal-week-1.md:14]: would like to see you grab one of these topics and expand/reflect on them or the learning process.

**Pull request merged**